src/main/java/pl/xkoem/GameBoard.java

```diff
@@ -0,0 +1,34 @@
+package pl.xkoem;
+
+import java.util.function.Consumer;
+
+public class GameBoard {
+
+    private Symbol[] boardSymbols;
+    private final Integer width;
+    private final Integer height;
+    private final Consumer<String> userOutput;
+
+
+    public GameBoard(GameConfiguration gameConfiguration, Consumer<String> userOutput) {
+        Integer[] boardSize = gameConfiguration.getBoardSize();
+        width = boardSize[0];
+        height = boardSize[1];
+        this.userOutput = userOutput;
+        Integer numberOfPositions = width * height;
+        boardSymbols = new Symbol[numberOfPositions];
```

Comment:
Jesteś pewny, że w przypadku planszy która będzie miała 1000x1000, a gra zakończy się w 3 turach chcesz trzymać tablicę z 1 000 000 (słownie milionem) elementów?

Reply:
Hmmm, na razie tak. Proszę dopisz mi to w issues.